src/streaming/controllers/BufferController.js

```diff
@@ -546,7 +545,9 @@ function BufferController(config) {
         eventBus.trigger(Events.BUFFER_CLEARED, {sender: instance, from: e.from, to: e.to, hasEnoughSpaceToAppend: hasEnoughSpaceToAppend()});
         if (hasEnoughSpaceToAppend()) return;
 
-        setTimeout(clearBuffer(getClearRange()), minBufferTime * 1000);
+        setTimeout(function () {
```

Comment:
I know it was not previously, but should we setTimeout to var so we can clear on reset if needed?